script/ping_websites.rb

```diff
@@ -49,6 +49,7 @@ def url_exist(name, url_string)
 rescue  Errno::EADDRNOTAVAIL,
         Errno::ECONNRESET,
         Errno::EHOSTUNREACH,
+        Errno::ENETUNREACH,
```

Comment:
This should be in a separate commit since it's not related to the CI improvements.